### PR TITLE
GlobalAlertTemplates for detectors: multivariable flow and dns tunnel

### PR DESCRIPTION
**Pull request by freecaykes**
## Description

Adding new detectors for multivariable flow and dns tunnel

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by tmjd — COMMENTED** [review]
Please have someone that can review the GlobalAlertTemplate review and approve this and then I'll be happy to approve. The PR looks good from an operator code perspective.

File: pkg/render/intrusion_detection.go [hunk 1]
```diff
@@ -1088,6 +1107,23 @@ func (c *intrusionDetectionComponent) adJobsGlobalertTemplates() []client.Object
...
+				Kind:       "GlobalAlertTemplate",
+				APIVersion: "projectcalico.org/v3",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: adDetectorPrefixName + "dns-tunnel",
+			},
+			Spec: v3.GlobalAlertSpec{
+				Type:        v3.GlobalAlertTypeAnomalyDetection,
+				Description: "DNS tunnel",
+				Summary:     "Looks for DNS queries names in the DNS log with subdomains that can contain encoded information.",
```
Comment (rene-dekker) [inline pkg/render/intrusion_detection.go:1121]: > Looks for DNS queries names

Grammatically incorrect.

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**